android/KMEA/app/src/main/assets/android-host.js

```diff
@@ -337,6 +337,14 @@ function menuKeyUp() {
   window.location.hash = hash;
 }
 
+// The keyboard-picker displayed via Android longpress disrupts Web-side
+// gesture-handling; this function helps force-clear the globe key's highlighting.
+function clearGlobeHighlight() {
+  if(keyman.osk && keyman.osk.vkbd && keyman.osk.vkbd.currentLayer.globeKey) {
+    keyman.osk.vkbd.currentLayer.globeKey.highlight(false)
+  }
```

Comment:
Is this a private API? If so, we need to document usage, ala

https://github.com/keymanapp/keyman/blob/f870e912f3bf0b36186a81e9bff912c0d02c8f74/developer/src/server/src/site/test.js#L238-L240

Reply:
@jahorton fyi